src/common/cockpitwebserver.c

```diff
@@ -1371,38 +1371,32 @@ cockpit_web_request_get_protocol (CockpitWebRequest *self)
 gchar *
 cockpit_web_request_get_remote_address (CockpitWebRequest *self)
 {
-  gchar *result = NULL;
```

Comment:
Hah, good one. Supposedly this was a leftover from before we had the g_auto* magic :grin: